pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -190,10 +190,12 @@ func TestConvertRbacRulesToFilterConfig(t *testing.T) {
 					{
 						Services: []string{"*"},
 						Constraints: []*rbacproto.AccessRule_Constraint{
-							{Key: "destination.port", Values: []string{"80", "443"}},
-							{Key: "destination.ip", Values: []string{"192.1.2.0/24", "2001:db8::/28"}},
-							{Key: "request.header[key1]", Values: []string{"prefix*", "*suffix"}},
-							{Key: "request.header[key2]", Values: []string{"simple", "*"}},
+							{Key: "port", Values: []string{"80", "443"}},
+							{Key: "ip", Values: []string{"192.1.2.0/24", "2001:db8::/28"}},
+							{Key: "header:key1", Values: []string{"prefix*", "*suffix"}},
+							{Key: "header:key2", Values: []string{"simple", "*"}},
+							{Key: "label:version", Values: []string{"v10"}},
+							{Key: "name", Values: []string{"attr-name"}},
```

Comment:
The "key" should be attribute names. Why are they changed to short strings like "port"/"ip"?

Reply:
This is the same reason to my last reply, plus one more reason that it's used in both ServiceRole.Constraints and ServiceRoleBinding.Properties, so either we need both `destination.ip`\`source.ip` or just make it `ip`, the place where it's used already tell us if it's a destination or source.